### PR TITLE
Start with Report Design perspective

### PR DESCRIPTION
**Pull request by hvbtup**
This starts the "all-in-one" designer and the RCP designer (which I actually never used) with the Report Design perspective.
The other topics mentionend in #694 (splash screen, icons, welcome page) are still TO DO.

### CODE REVIEW

**Review by gab1one — APPROVED** [review]

File: build/birt-packages/birt-report-all-in-one/BIRT.product [hunk 1]
```diff
@@ -7,7 +7,10 @@
    </configIni>
 
    <launcherArgs>
-      <vmArgs>-Xms512m
+      <programArgs>-perspective
+org.eclipse.birt.report.designer.ui
```
Comment (ruspl-afed) [inline build/birt-packages/birt-report-all-in-one/BIRT.product:11]: I cannot find the perspective with identifier `org.eclipse.birt.report.designer.ui`
Please double check that we have the definition with such an identifier.

File: build/birt-packages/birt-report-rcp/BIRT.product [hunk 2]
```diff
@@ -7,13 +7,18 @@
    </configIni>
 
    <launcherArgs>
-      <vmArgs>-Xms512m
+      <programArgs>-perspective
+org.eclipse.birt.report.designer.ui
```
Comment (ruspl-afed) [inline build/birt-packages/birt-report-rcp/BIRT.product:11]: I cannot find the perspective with identifier `org.eclipse.birt.report.designer.ui`
Please double check that we have the definition with such an identifier.

**Review by ruspl-afed — COMMENTED** [review]
@hvbtup  I have concerns regarding the perspective identifer
if this PR is "work in progress" please mark it accordingly

**Comment by hvbtup** [comment]
Somehow my comment got lost.
To put a long story short: Finding the ID was more or less a guess. It seemed to work for me, but that was only because I used an existing .metadata. The way to find the perspective ID described in #694 did not work. I found another way on https://stackoverflow.com/questions/3383470/how-to-get-an-id-perspective-of-eclipse/, re-tested with a very clean start script and confirmed that ID org.eclipse.birt.report.designer.ui.ReportPerspective is working as expected.

**Comment by hvbtup** [comment]
Not sure if the same ID is correct for the RCP Designer, though. I don't know how the RCP Designer should look, because I never used it before.

**Comment by wimjongman** [comment]
This was fixed with #900

**Pull request closed without merging**